ui/store/actions.test.js

```diff
@@ -1619,6 +1619,31 @@ describe('Actions', () => {
     });
   });
 
+  describe('#setParticipateInMetaMetrics', () => {
+    beforeAll(() => {
+      window.sentry = {
+        toggleSession: jest.fn(),
+        endSession: jest.fn(),
+      };
+    });
+    it('sets participateInMetaMetrics to true', async () => {
+      const store = mockStore();
+      const setParticipateInMetaMetricsStub = sinon
+        .stub()
+        .callsFake((_, cb) => cb());
```

Comment:
can we use jest here as well?

Reply:
Id rather do that as a larger part of fixing the actions test file to remove sinon. I copied patterns used for other background methods here. Do we have an issue to track dropping sinon?

Reply:
Mmm... i might retract my statement. let me give it a stab and see how it looks

Reply:
added the jest version and removed the additional sinon usage.